### DISCUSSION TITLE
Missing Icons

### DISCUSSION THREAD

**Discussion by coffeeandubuntu** [Jyers/Marlin#648] [Q&A] [answered]
I flashed the latest firmware (1.3.1 -- E3V2-Default-4.2.2) but am missing icons on the bottom of the main screen. Any ideas on how to fix? Do I need to flash my screen? 

![image](https://user-images.githubusercontent.com/83423213/116596352-3e8e5780-a8f2-11eb-893d-08558382ce33.png)

**Comment by gothcha** [comment, marked as answer]
Which icons do you think are missing? - it looks fine to me.

**Reply by coffeeandubuntu** [reply]
Sorry... I should have been more clear. That was a screenshot of how the screen should look. On my machine, all the icons on that screenshot are missing. I just see numbers. It looks like this:

![image](https://user-images.githubusercontent.com/83423213/116597812-0e47b880-a8f4-11eb-9640-30ed2dc08deb.png)

**Reply by tome9111991** [reply]
You can try update Display

**Reply by coffeeandubuntu** [reply]
> You can try update Display

Not trying to be dense, but do you have a description for how I would do that?

I ask because the Wiki has this note: "Note: Screen flashing will never be required, custom icons will simply be substituted with existing ones if you have not flashed your screen." That seems to suggest I don't need to flash my screen.

**Reply by Timbo200** [reply]
Prior to using Jyers' firmware did you have the most recent Creality screen firmware?

**Reply by coffeeandubuntu** [reply]
I fixed the issue! Just needed to power-cycle the printer. Big thank you to Timbo200, tome9111991, and Gothcha for responding so quickly!

This firmware looks awesome!

**Reply by gothcha** [reply]
Great enjoy👍
